operator/apis/loki/register.go

```diff
@@ -0,0 +1 @@
+package loki
```

Comment:
Is this needed for the docs-generator to pick up the directory as a package? Does it make sense to add a package comment?

Reply:
yes it is needed for the doc generator